app/ui/src/app/customizations/api-connector/api-connector-list.component.ts

```diff
@@ -58,7 +66,23 @@ export class ApiConnectorListComponent implements OnInit {
     };
   }
 
+  getAppName() {
+    this.appName = this.config.getSettings('branding', 'appName', 'Syndesis');
+  }
+
+  handleAction(event: any) {
+    if (event.id === 'createApiConnector') {
+      this.router.navigate(['create'], { relativeTo: this.route });
+    }
+  }
+
+  handleClick(event: any) {
+    const apiConnector = event.item;
+    this.router.navigate([apiConnector.id], { relativeTo: this.route });
+  }
+
   ngOnInit() {
-    this.store.loadAll();
+    this.getAppName();
```

Comment:
the `getAppName()` method is used only once and we do not expect to execute it outside the component domain (it is not even possible, since the component API is not exported thru an `exportAs` property in the decorator).

We also only need to initialize `this.appName` just once, so I'd remove the `getAppName()` method and insert its implementation here, like this:

```typescript
ngOnInit() {
  this.appName = this.config.getSettings('branding', 'appName', 'Syndesis');
  this.apiConnectorStore.loadAll();
}
``` 

the lesser the code, the better ;)

Reply:
Sure thing!